driver/pelican/main.go

```diff
@@ -14,6 +14,7 @@ import (
 
 const TSTAT_PO_DF = "2.1.1.0"
 const DR_PO_DF = "2.1.1.9"
+const OCCUPANCY_PO_DF = "2.1.2.1"
```

Comment:
I'm just using the existing `i.xbos.occupancy` interface. Let me know if I should create a different interface to use here.

Reply:
LGTM!